src/jit/v.lua

```diff
@@ -86,10 +86,28 @@ local function fmtfunc(func, pc)
   end
 end
 
+local function vmdef_name(id, message)
+  local targetTable;
+  if string.find(message, "bytecode %s", 1, true) then
```

Comment:
I used string.find instead of global enums as performance doesn't matter here.

We could also get buildvm.c to generate a hashtable like this : 

```Lua
local TREDEF = {
	RECERR = "error thrown or hook called during recording",
	TRACEUV = "trace too short",
	TRACEOV = "trace too long",
	STACKOV = "trace too deep",
	SNAPOV = "too many snapshots",
	BLACKL = "blacklisted",
	RETRY = "retry recording",
	NYIBC = "NYI: bytecode %s",
	LLEAVE = "leaving loop in root trace",
	LINNER = "inner loop in root trace",
	LUNROLL = "loop unroll limit reached",
	BADTYPE = "bad argument type",
	CJITOFF = "JIT compilation disabled for function",
	CUNROLL = "call unroll limit reached",
	DOWNREC = "down-recursion, restarting",
	NYIFFU = "NYI: unsupported variant of FastFunc %s",
	NYIRETL = "NYI: return to lower frame",
	STORENN = "store with nil or NaN key",
	NOMM = "missing metamethod",
	IDXLOOP = "looping index lookup",
	NYITMIX = "NYI: mixed sparse/dense table",
	NOCACHE = "symbol not in cache",
	NYICONV = "NYI: unsupported C type conversion",
	NYICALL = "NYI: unsupported C function type",
	GFAIL = "guard would always fail",
	PHIOV = "too many PHIs",
	TYPEINS = "persistent type instability",
	MCODEAL = "failed to allocate mcode memory",
	MCODEOV = "machine code too long",
	MCODELM = "hit mcode limit (retrying,",
	SPILLOV = "too many spill slots",
	BADRA = "inconsistent register allocation",
	NYIIR = "NYI: cannot assemble IR instruction %s",
	NYIPHI = "NYI: PHI shuffling too complex",
	NYICOAL = "NYI: register coalescing too complex"
}
```

And then i could directly do : 

```lua
if message == vmdef.TREDEF.NYIBC then
    ...
elseif ...
```

or even link it to another table that handle the redirection

Reply:
fixed in b2596b32f17504beb2027d06cbf6be1bbe81436d